controllers/operatorconfigmap_controller.go

```diff
@@ -91,19 +92,20 @@ func (c *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		),
 	)
 	// Reconcile the ClusterVersion object when the operator config map is updated
-	enqueueClusterVersionRequest := handler.EnqueueRequestsFromMapFunc(
+	enqueueConfigMapRequest := handler.EnqueueRequestsFromMapFunc(
 		func(_ context.Context, client client.Object) []reconcile.Request {
 			return []reconcile.Request{{
 				NamespacedName: types.NamespacedName{
-					Name: clusterVersionName,
+					Name:      operatorConfigMapName,
+					Namespace: client.GetNamespace(),
```

Comment:
@bernerhat As I already explined in a prev comment here: https://github.com/red-hat-storage/ocs-client-operator/pull/64#discussion_r1470035357

using `client.GetNamespace` is wrong! client in this case is the object that produces the event. In our use case, this will be the ClusterVersion CR which is a cluster-wide CR and has no namespace associated with it. But even if it was a namespaced CR, the namespace most likely would be different.  

You need to use the operator's namespace not the object's namespace

Reply:
changed into `c.OperatorNamespace` is that acceptable?